browser/ui/webui/brave_settings_ui.cc

```diff
@@ -236,3 +235,19 @@ void BraveSettingsUI::BindInterface(
   assistant_handler->BindInterface(std::move(pending_receiver));
   web_ui()->AddMessageHandler(std::move(assistant_handler));
 }
+
+void BraveSettingsUI::BindInterface(
+    mojo::PendingReceiver<brave_account::mojom::BraveAccountHandlerFactory>
+        pending_receiver) {
+  if (brave_account_handler_factory_receiver_.is_bound()) {
+    brave_account_handler_factory_receiver_.reset();
+  }
```

Comment:
Maybe this should just `CHECK(!brave_account_handler_factory_receiver_.is_bound())`